src/components/Upgrade/content.js

```diff
@@ -76,13 +76,13 @@ export const highlights = [
   {
     title: {
       small: null,
-      large: 'Update apps from the App Center',
+      large: 'Update apps from Apps',
     },
     description: {
       small: null,
       large: `
         Apply security patches and add new features to the apps in your
-        organization by updating to the latest version of apps in the App Center.
+        organization by updating to the latest version of apps in Apps.
```

Comment:
We may want to rethink the copy here cc @dizzypaty, "upgrade apps from/in Apps" does not sound very clear 😄